frontend/services/books.ts

```diff
@@ -0,0 +1,36 @@
+import { getSession } from './session'
+
+const REST_SERVER = process.env.REST_SERVER || ''
+
+export type Book = {
+  id: string
+  title: string
+  description: string
+}
+
+export type ListBooksType = {
+  items: Book[]
+  pages: number
+  current_page: number
+}
+
+export const listBooks = async (): Promise<ListBooksType> => {
+  const session = await getSession()
+  const response = await fetch(`${REST_SERVER}/api/v1/books`, {
+    method: 'GET',
+    headers: {
+      Accept: 'application/json',
+      'Content-Type': 'application/json',
+      Authorization: `bearer ${session.accessToken}`
```

Comment:
`getSession` will either return a string value or null. How will this handle if session is null?

Reply:
Oh nice! Ayts, will catch this. Thanks!

Reply:
@normancapule  Removed the use the session fetcher.

I am now able to work with `unstable_getServerSession` of `next-auth/next`